### PR TITLE
Bug fix for exposed ports on copy

### PR DESCRIPTION
**Pull request by msDestiny14**
https://github.com/BabylonJS/Babylon.js/issues/11049

### CODE REVIEW

File: nodeEditor/src/diagram/graphFrame.ts [hunk 1]
```diff
@@ -1435,6 +1435,26 @@ export class GraphFrame {
         StringTools.DownloadAsFile(state.hostDocument, json, this._name + ".json");
     }
 
+    public adjustPorts() {
+        for (var node of this.nodes) {
+            for (var port of node.outputPorts) { // Output
```
Comment (RaananW) [inline nodeEditor/src/diagram/graphFrame.ts:1440]:
```suggestion
        for (let node of this.nodes) {
            for (let port of node.outputPorts) { // Output
```

File: nodeEditor/src/diagram/graphFrame.ts [hunk 2]
```diff
@@ -1435,6 +1435,26 @@ export class GraphFrame {
...
+        for (var node of this.nodes) {
+            for (var port of node.outputPorts) { // Output
+                if (port.exposedOnFrame) {
+                    if (port.exposedPortPosition !== -1) {
+                        this._exposedOutPorts[port.exposedPortPosition] = port;
+                    }
+                }
+            }
+
+            for (var port of node.inputPorts) { // Inports
```
Comment (RaananW) [inline nodeEditor/src/diagram/graphFrame.ts:1448]:
```suggestion
            for (let port of node.inputPorts) { // Imports
```

**Review by RaananW — APPROVED** [review]
nits :-)

File: nodeEditor/src/diagram/graphFrame.ts (hunk 1 above)
Comment (PolygonalSun) [inline nodeEditor/src/diagram/graphFrame.ts]: Nit: Use `const` or `let` instead of `var`

**Review by RaananW — COMMENTED** [review]
.

File: nodeEditor/src/diagram/graphFrame.ts (hunk 2 above)
Comment (PolygonalSun) [inline nodeEditor/src/diagram/graphFrame.ts]: Nit: Use `const` or `let` instead of `var`

**Pull request merged**